src/data/flight.json

```diff
@@ -1506,16 +1506,16 @@
     "reportState": "SA",
     "flightNo": "EY84",
     "airline": "Etihad Airways",
-    "path": "Rome � Abu Dhabi",
+    "path": "Rome to Abu Dhabi",
     "dateDeparture": "",
     "dateArrival": "09-03-20",
-    "closeContactRow": "29, 30, 31, 32, 33, 34, 35, 36"
+    "closeContactRow": "29, 30, 31, 32, 33, 34, 35,36"
```

Comment:
this line needs to be modified

Reply:
Thank you! I have changed and pushed to my branch.